src/bindings/io/doc/elektra-io-doc.pc.in

```diff
@@ -0,0 +1,14 @@
+prefix=@CMAKE_INSTALL_PREFIX@
```

Comment:
Is this file needed?

Reply:
No. At first I thought of it as a template for new bindings, but I removed unnecessary files. The binding is for documentation.